### PR TITLE
(745) Fetch keyworkers from reference data endpoint

### PR DESCRIPTION
**Pull request by pezholio**
This removes the hardcoded keyworkers, and instead fetches them from the `/reference-data/key-workers` endpoint.

### CODE REVIEW

File: wiremock/stubs/keyworkers.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  { "id": "720cbcb6-2eb8-4f10-90a0-a8e8589a6efc", "name": "Reuben Konopelski", "isActive": true },
+  { "id": "a741d453-796d-476b-8d27-5b4a1d1e0d09", "name": "Denae Spencer", "isActive": true },
+  { "id": "99f6122d-a0c4-47eb-8eb0-43809a384eaa", "name": "Mozella Russel", "isActive": true },
+  { "id": "0e67af2b-c93e-48a1-9dc5-2a08e0d4d82c", "name": "Willene Towne", "isActive": true },
+  { "id": "0e29769d-f883-4b6b-85de-1b014fc125df", "name": "Randell Adams DDS", "isActive": true },
+  { "id": "7b41acdd-1e66-4cfc-91e1-47ab40a70694", "name": "Dr. Alexander Gibson", "isActive": true },
+  { "id": "0cc91821-3a95-4cf5-bbdd-8d44124e01f3", "name": "Cammy McDermott", "isActive": true },
+  { "id": "1889b41b-0850-4fba-ac4a-f1bf07ae7201", "name": "Carmen Walter", "isActive": true },
+  { "id": "0895532a-6c62-462c-96e9-5b545d70310c", "name": "Van Batz", "isActive": true },
+  { "id": "6179471a-776d-4972-99c4-89c5679c4253", "name": "Dr. Benedict Kunde", "isActive": true }
```
Comment (richpjames) [inline wiremock/stubs/keyworkers.json:21]: Is it worth having one with `isActive` set to false?

File: server/services/bookingService.test.ts [hunk 2]
```diff
@@ -72,6 +80,26 @@ describe('BookingService', () => {
     })
   })
 
+  describe('getKeyworkers', () => {
+    it('should return the keyworker data needed', async () => {
+      const keyWorkers = referenceDataFactory.buildList(2)
+
+      referenceDataClient.getReferenceData.mockImplementation(category => {
```
Comment (richpjames) [inline server/services/bookingService.test.ts:87]: We could just `.mockResolvedValue(keyWorkers)` here? Or am I missing something?

File: server/services/bookingService.test.ts [hunk 3]
```diff
@@ -72,6 +80,26 @@ describe('BookingService', () => {
     })
   })
 
+  describe('getKeyworkers', () => {
+    it('should return the keyworker data needed', async () => {
+      const keyWorkers = referenceDataFactory.buildList(2)
```
Comment (richpjames) [inline server/services/bookingService.test.ts:85]: if we're to be consistent with the name of the method this should be `keyworkers`

File: server/controllers/manage/bookingsController.test.ts [hunk 4]
```diff
@@ -51,42 +52,52 @@ describe('bookingsController', () => {
 
   describe('new', () => {
     describe('If there is a CRN in the flash', () => {
-      it('it should render the new booking form', async () => {
-        const person = personFactory.build()
+      const person = personFactory.build()
+      const keyWorkers = referenceDataFactory.buildList(5)
+
+      beforeEach(() => {
```
Comment (richpjames) [inline server/controllers/manage/bookingsController.test.ts:58]: 👌 lovely

**Review by richpjames — APPROVED** [review]
Looks good, left a couple comments

File: server/services/bookingService.test.ts (hunk 2 above)
Comment (pezholio) [reply server/services/bookingService.test.ts]: Yeah, probs a bit overengineered!

File: server/services/bookingService.test.ts (hunk 3 above)
Comment (pezholio) [reply server/services/bookingService.test.ts:85]: I've renamed the method, as the endpoint is `key-workers`

**Pull request merged**